client/my-sites/sharing/connections/connection.jsx

```diff
@@ -204,6 +203,7 @@ export default connect(
 			siteId,
 			userHasCaps: canCurrentUser( state, siteId, 'edit_others_posts' ),
 			userId: getCurrentUserId( state ),
+			getUser: userId => UsersStore.getUser( getSelectedSiteId( state ), userId ),
```

Comment:
I'd prefer that we not pass through a function. Perhaps something similar to the following:
```jsx
( state, ownProps ) => {
//...
const userId = get( ownProps, 'connection.keyring_connection_user_ID', null );
const keyringUser = userId ? UserStore.getUser( null );
```